### ISSUE TITLE
Rabian Jessie support

### ISSUE THREAD

**Issue by riodda** [radxa/apt#5]
Hi, i've found the Rock and Rock Pro have been completely abandoned, i would be nice if some good smaritan could build a recent image for those boards, they are more than capable of running something more modern than jessie, or at least would be nice to add Jessie Support.
Thanks.

**Comment by Utano** [comment]
+1 :)

**Comment by MichaIng** [comment]
The generic Debian kernel actually supports this board: https://packages.debian.org/bullseye/linux-image-5.9.0-4-armmp
See the file list, which contains `/usr/lib/linux-image-5.9.0-4-armmp/rk3188-radxarock.dtb`, i.e. the Radxa Rock device tree.
Sadly there is no matching U-Boot bootloader package. That would need to be build, as I guess the old [Rockchip bootloader](https://wiki.radxa.com/Rock/bootloader) or [U-Boot](https://wiki.radxa.com/Rock/U-Boot) version provided by Radxa does not support device trees?

**Comment by riodda** [comment]
Package that is way over my linux skill ! 
It's sad that Radxa don't support they old board even the hardware is still decent, frankly is very unlikely i will buy again from them if this is the behaviour.

**Comment by MichaIng** [comment]
> It's sad that Radxa don't support they old board

That is pretty standard across SBC manufacturers (in a way similar to Android phone manufacturers) 😉. The sold pieces count is simply not high enough to financially allow ongoing kernel and image development, that is a Raspberry Pi privilege. The only thing I am not sure about is why manufacturers do not implement their drivers into mainline Linux right from the start, so that e.g. a regular Debian image with official Debian kernel can be offered, making an own repository obsolete. But I guess the Linux mainline merging + time until that Linux version reaches (especially) Debian stable repository (Arch and such rolling release distributions are a different case) is simply too long. So an own fork is created and effort bound to it for as long as reasonable.

Probably with ongoing Debian Bullseye development, the installer will support that board natively, so that you can flash the Debian installer onto a USB stick, boot the Radxa Rock from that and install Debian (including the mentioned kernel) onto the internal SD card or other drive that shall be used as root+boot file system.

**Comment by jack-ma** [comment]
images for rock/rock_pro:

https://forum.radxa.com/t/rock-and-rock2-rk3188-and-rk3288/3742/15

rk3188 is not listed as open source soc by Rockchip and the kernel is not maintained by Rockchip any more. We rely on the community to support them.

**Comment by MichaIng** [comment]
> rk3188 is not listed as open source soc by Rockchip

Good point, SoC manufacturer != SBC manufacturer. Support by mainline kernel has hence been backwards-engineered or taken from other Rockchip SoC drivers?
